src/domains/ecommerce-app/env/weu-dev/terraform.tfvars

```diff
@@ -37,3 +37,19 @@ ecommerce_xpay_psps_list = "testPSP1,testPSP2"
 ecommerce_vpos_psps_list = "testPSP3,testPSP4"
 
 dns_zone_checkout = "dev.checkout"
+
+cidr_subnet_ecommerce_functions = "10.1.155.0/24"
```

Comment:
pay attention to subnets, this is used by afm storage https://github.com/pagopa/pagopa-infra/blob/5c5b1573b35b64624317b0dbe1ba1f4cd15eaf35/src/domains/afm-common/env/weu-dev/terraform.tfvars#L55

Reply:
Whoops! Fixed in bcf8f6ba